ftplugin/ps1.vim

```diff
@@ -11,6 +11,8 @@ if exists("b:did_ftplugin") | finish | endif
 " Don't load another plug-in for this buffer
 let b:did_ftplugin = 1
 
+compiler psake
```

Comment:
Quite a few assumptions are made here about the script, so I don't think we should default the compiler. By far, I would expect the majority of scripts to be just normal scripts psake would fail to execute. Besides, @PProvost has generally pushed back on putting any general user preferences in these files (and I generally agree) and let them define how they want them set. We could maybe add tips to the README.md, but even that may be outside the purview of this project.